saleor/graphql/notifications/tests/test_external_notification_query.py

```diff
@@ -0,0 +1,156 @@
+
+query_test_data = [
+    (
+        {
+            "input": {
+                "ids": [],
+                "extraPayload": json.dumps("{}"),
+                "externalEventType": {},
+                "channel": "c-pln",
+            },
+            "pluginId": "",
+        },
+        200,
+    ),
+    (
+        {
+            "input": {
+                "ids": [],
+                "extraPayload": json.dumps("{}"),
+                "externalEventType": {},
+                "channel": "c-pln",
+            },
+            "pluginId": "WRONG-TEST-PLUGIN",
+        },
+        200,
+    ),
+    (
+        {
+            "input": {
+                "ids": [],
+                "extraPayload": json.dumps("{}"),
+                "externalEventType": {},
+                "channel": "c-pln",
+            }
+        },
+        200,
+    ),
+    (
+        {
+            "input": {
+                "ids": [],
+                "extraPayload": json.dumps("{}"),
+                "externalEventType": {},
+            }
+        },
+        400,
+    ),
+    (
+        {
+            "input": {
+                "extraPayload": json.dumps("{}"),
+                "externalEventType": {},
+                "channel": "c-pln",
+            }
+        },
+        400,
+    ),
+    ({"input": {"ids": [], "externalEventType": {}, "channel": "c-pln"}}, 200),
+    (
+        {
+            "input": {
+                "ids": [],
+                "extraPayload": json.dumps("{}"),
+                "channel": "c-pln",
+            }
+        },
+        400,
+    ),
+]
+
+
+@pytest.mark.parametrize("variables, status_code", query_test_data)
+def test_query(
+    variables,
+    status_code,
+    external_notification_trigger_query,
+    staff_api_client,
+    permission_manage_users,
+    channel_PLN,
+):
+    response = staff_api_client.post_graphql(
+        external_notification_trigger_query,
+        variables,
+        permissions=[permission_manage_users],
+        check_no_permissions=False,
+    )
+    assert response.status_code == status_code
+
+
+@patch("saleor.plugins.manager.PluginsManager.notify")
+def test_notify_via_external_notification_trigger_for_all_plugins(
+    plugin_manager_notify_mock,
+    settings,
+    staff_users,
+    external_notification_trigger_query,
+    staff_api_client,
+    permission_manage_users,
+    channel_PLN,
+):
+
+    settings.PLUGINS = ["saleor.plugins.tests.sample_plugins.PluginSample"]
+
+    variables = {
+        "input": {
+            "ids": [to_global_id(User.__name__, user.id) for user in staff_users],
+            "extraPayload": "{}",
+            "externalEventType": UserNotifyEvent.ORDER_CANCELED,
+            "channel": channel_PLN.slug,
+        }
+    }
+
+    response = staff_api_client.post_graphql(
+        external_notification_trigger_query,
+        variables,
+        permissions=[permission_manage_users],
+    )
+
+    assert response.status_code == 200
```

Comment:
I think it doesn't mean anything. If you want to be sure that your mutation doesn't returned an error, then you should confirm that returned payload doesn't have field `errors`. (You can find a lot of examples in Saleor's tests.)